WebRoot/js/ajax/util/AjxDateUtil.js

```diff
@@ -283,21 +283,22 @@ function(date) {
 }
 
 AjxDateUtil.computeDateStr =
-function(now, dateMSec) {
+function(now, dateMSec, requireTime) {
 	if (dateMSec == null)
 		return "";
 
 	var date = new Date(dateMSec);
+	var time = AjxDateUtil.computeTimeString(date);
 	if (now.getTime() - dateMSec < AjxDateUtil.MSEC_PER_DAY &&
 		now.getDay() == date.getDay()) {
-		return AjxDateUtil.computeTimeString(date);
+		return time;
 	}
 
 	if (now.getFullYear() == date.getFullYear()) {
-		return AjxDateUtil._dateFormatNoYear.format(date);
+		return AjxDateUtil._dateFormatNoYear.format(date) + (requireTime ? " " + time : "");
```

Comment:
I am not sure if there could be some languages which wants to jumble up sequence of date and time parts, if that is the case this logic will fail

Reply:
Can we use different formatters in case when we want to show time also?

Reply:
@silentsakky I will change it to use another (new?) formatter.